libs/newsletter/driver/hubspot/src/newsletter.service.spec.ts

```diff
@@ -1,42 +1,72 @@
-import { TestBed } from '@angular/core/testing';
 import {
-  cold,
-  hot,
-} from 'jasmine-marbles';
-import { Observable } from 'rxjs';
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
 
 import { DaffNewsletterHubspotService } from './newsletter.service';
 import { DAFF_NEWSLETTER_HUBSPOT_FORMS_TOKEN } from './token/hubspot-forms.token';
+import { DaffNewsletterSubmission } from '@daffodil/newsletter';
+import {
+  DaffNewsletterDriver,
+  DaffNewsletterServiceInterface,
+} from '@daffodil/newsletter/driver';
+import { DaffNewsletterHubSpotDriverModule } from '@daffodil/newsletter/driver/hubspot';
+import { Observable } from "rxjs";
+import { hot } from "jasmine-marbles";
+import { HubspotResponse } from '@daffodil/driver/hubspot';
 
-describe('DaffNewsletterHubspotService', () => {
-  let newsletterService;
+describe('DaffNewsletterHubspotDriver', () => {
+  let newsletterService: DaffNewsletterServiceInterface;
+  let httpMock: HttpTestingController;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
+      imports: [
```

Comment:
This should not use the `HttpClientTestingModule`. You should either expose a mock of the driver of the `@daffodil/driver/hubspot/testing` package or just make a simple mock here.